docs/src/global/componentApiTable.jsx

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable react/no-danger */
+
+// import './ComponentApiTable.scss';
+
+import {
+    isEmpty,
+    map,
+} from 'lodash';
+import { Table } from 'react-cm-ui';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { DOCS_PROPS_PROP_TYPE } from './componentApiConstants';
+
+const propTypes = {
+    componentProps: DOCS_PROPS_PROP_TYPE.isRequired,
+    style: PropTypes.shape({}),
+};
+
+const defaultProps = {
+    style: {},
+};
+
+function getTypeName(type) {
+    let allowedTypesString;
+    let unionString;
+    let arrayOfString;
+    let shapeString;
+
+    // console.log('type', type);
```

Comment:
Removing `console.log`

Reply:
Looks like the entire function containing this commented out `console.log()` was removed in db1eed1.  Resolving.